person-service/src/main/kotlin/com/thehuginn/controller/AddressResource.kt

```diff
@@ -0,0 +1,37 @@
+package com.thehuginn.controller
+
+import com.thehuginn.controller.dto.response.AddressResponse
+import com.thehuginn.controller.dto.response.mapper.AddressResponseMapper
+import com.thehuginn.service.AddressService
+import jakarta.ws.rs.GET
+import jakarta.ws.rs.Path
+import jakarta.ws.rs.QueryParam
+import java.util.UUID
+
+@Path("/address")
+class AddressResource(
+    private val addressService: AddressService,
+    private val addressResponseMapper: AddressResponseMapper
+) {
+
+    @GET
+    @Path("hello")
+    fun hello(): String {
+        return "Hello"
+    }
+
+    @GET
+    @Path ("/addressById")
```

Comment:
toto nechaj povodne pls, teda "/{id}" a ten druhy nechaj bez @Path a iba s query param. Takto to nie je uplne RESTovsky pristup